src/pages/index.tsx

```diff
@@ -63,16 +66,19 @@ const HomePage: NextPage<HomePageProps> = ({
   )
 }
 
-export const getStaticProps: GetStaticProps = async () => {
+export const getServerSideProps: GetServerSideProps = async () => {
   // @TODO: get this from a rest api sometime in the future
   const meta = await import('../content/meta').then((m) => m.default)
 
   // @TODO: get this from a rest api sometime in the future
-  const cases = await import('../content/cases').then((m) => m.default)
+  const cases = await client.items.all(
```

Comment:
can we move this into a separate file? maybe something like `lib/datocms`, and export a function named `fetchCases`, annotated with TypeScript returning `Case[]`